python_tests/pytest_input_generators.py

```diff
@@ -450,6 +455,118 @@ def index_select_error_generator(
     # yield SampleInput(a, b, 0), RuntimeError, "out of bounds index value."
 
 
+def pad_error_generator(
+    op: OpInfo, dtype: torch.dtype, requires_grad: bool = False, **kwargs
+):
+    # Nvfuser - fd.ops.pad(Tensor arg, std::vector<int64_t>& pad_widths, std::optional<Scalar> value)
```

Comment:
Shouldn't we remove `std::optional` from this line if pad value is no longer optional?

Reply:
I added back the optional value.